docs/repo-specific-configuration.md

```diff
@@ -32,6 +32,11 @@ You can add `<YOUR_REPO>/.scala-steward.conf` to configure how Scala Steward upd
 #pullRequests.frequency = "0 0 ? * 3" # every thursday on midnight
 pullRequests.frequency = "7 days"
 
+# pullRequests.labelRegex allows to control which labels are added to PRs 
+# via a regex check each label is checked against.
+# Defaults to no regex (all labels are added) which is equivalent to ".*".
+pullRequests.labelRegex = "(.*semver.*)|(commit-count:n:.*)"
```

Comment:
I think it would be better naming if the name indicate its behavior or use case.
Could you consider different nam like `pullRequests.excludeMatchedLabels` or such 🙇

Reply:
👍 It's inclusive, not exclusive though, so I went for `pullRequests.includeMatchedLabels`